cpu/esp_common/Makefile.dep

```diff
@@ -98,6 +98,10 @@ ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
   USEMODULE += ztimer_msec
 endif
 
+ifneq (,$(filter esp_eth,$(USEMODULE)))
```

Comment:
`esp_wifi` and `esp_now` too, right?

Reply:
They actually already set it a few lines above. But let's refactor it a bit to make this less convoluted.